### ISSUE TITLE
Make buttons with inline icons style properly

### ISSUE THREAD

**Issue by huayunh** [brightlayer-ui/angular-themes#31]
Buttons with icons in them have always been a huge problem to get the styling correct. So why not bring it to our themes?

Maybe for `<mat-icon>` with an `inline` attribute:

```html
<button mat-stroked-button color="primary">
    <mat-icon inline>favorite</mat-icon>
    Inline Icon
</button>
```

instead of styling it like 

![image](https://user-images.githubusercontent.com/8997218/134556351-54f1a1dc-1bea-4389-85fa-cda40f9cf9c9.png)

They should look like [what Material Design suggests](https://material.io/components/buttons#specs)

![image](https://user-images.githubusercontent.com/8997218/134556459-0f67e34c-ffc8-4264-98f4-8f27e07d33f3.png)

- Ignore the `min-width` part, just focus on the margins
- also make icon size 20x20 to match React

----
### Additional context
- This would be a breaking change
- Do not touch `<mat-icon>` without the `inline` attribute — they are used for regular icon buttons and you wouldn't want to mess up the style

**Comment by joebochill** [comment]
Agree that it would be preferable to fix this in Angular Material instead of our themes. Let's log an issue (make a PR) there instead.

**Comment by emclaug2** [comment]
Fixed in #65
